extras/nest_vars.sh.in

```diff
@@ -1,26 +1,19 @@
 #!/bin/sh
 
-# NEST is installed here. When you relocate NEST, change this variable.
-export NEST_INSTALL_DIR="@CMAKE_INSTALL_PREFIX@"
+# NEST is installed here. When you relocate NEST, change these variables.
+NEST_LIB_DIR="@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_LIBDIR@/nest/"
+NEST_BIN_DIR="@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_FULL_BINDIR/"
 
-# NEST finds standard *.sli files $NEST_DATA_DIR/sli
-export NEST_DATA_DIR="${NEST_INSTALL_DIR}/@CMAKE_INSTALL_DATADIR@"
-
-# NEST finds help files $NEST_DOC_DIR/help
-export NEST_DOC_DIR="${NEST_INSTALL_DIR}/@CMAKE_INSTALL_DOCDIR@"
-
-# The path in which NEST dynamic libraries are found. Works also if $(DY)LD_LIBRARY_PATH is undefined.
+# The path in which NEST dynamic libraries are found.
+# Works also if $(DY)LD_LIBRARY_PATH is undefined.
 if [[ "$OSTYPE" == "darwin"* ]]; then
-    export DYLD_LIBRARY_PATH="$NEST_INSTALL_DIR/@CMAKE_INSTALL_LIBDIR@/nest"${DYLD_LIBRARY_PATH:+:$DYLD_LIBRARY_PATH}
+    export DYLD_LIBRARY_PATH="$NEST_LIB_DIR"${DYLD_LIBRARY_PATH:+:$DYLD_LIBRARY_PATH}
```

Comment:
Why is the closing `"` here directly after `$NEST_LIB_DIR` and not at the very end as for all other paths set in this script?

Reply:
This is a bit of a minefield. This answer on StackOverflow is very enlightening: https://stackoverflow.com/questions/33318499/should-i-use-quotes-in-environment-path-names

I checked several combinations of quoted variable names, quotes-in-values and spaces-in-values, and couldn't break Bash in any scenario: quoting the whole right-hand side, or quoting only the variable name. However, for strict POSIX compliance we should quote the entire right-hand side, because when quoting only the variable name, /bin/sh does indeed break.

```suggestion
    export DYLD_LIBRARY_PATH="$NEST_LIB_DIR${DYLD_LIBRARY_PATH:+:$DYLD_LIBRARY_PATH}"
```